test/factories/api_docs/service.rb

```diff
@@ -12,5 +13,13 @@
         paths: {}
       }.to_json
     end
+
+    before(:validation) do |api_docs_service|
+      account = FactoryBot.create(:simple_provider)
+
+      api_docs_service.account = account
+      api_docs_service.owner = account
```

Comment:
I assume this won't overwrite if you pass on `FactoryBot.build(:api_docs_service, owner: object)`, right?

Reply:
actually, it does.